src/ROUTES.js

```diff
@@ -81,6 +95,19 @@ export default {
     WORKSPACE_INVITE: 'workspace/:policyID/invite',
     REQUEST_CALL: 'request-call',
 
+    /**
+     * This links to a page in e.com ensuring the user is logged in.
+     * It does so by getting a validate code and redirecting to the validate URL with exitTo set to the URL
+     * we want to visit
+     * @param {string} url relative URL to open in expensify.com
```

Comment:
Does "relative" mean "with a leading slash", if so, can you add that explicitly?